Contributing/Issue_Priorities.md

```diff
@@ -0,0 +1,32 @@
+<!---
+Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+## AMP GitHub Issue Priorities
+The AMP team is introducing the below priorities and guidelines to make it easier to order issues by the level of attention they need. 
+These new guidelines can give you also a good overview of when to expect updates or closure of issues.
+
+In order to add a priority to your issues use our [list of GitHub labels](https://github.com/ampproject/amphtml/labels).
```

Comment:
Are we expecting filers to set their own priority on issues?  If so, we should set the expectation that they are hints, but that there is a process for adjusting the priorities.  (We should also be very transparent for how those priorities are set.)